packages/snap-client/src/Client/apis/Recommend.test.ts

```diff
@@ -132,54 +132,56 @@ describe('Recommend Api', () => {
 		requestMock.mockReset();
 	});
 
-	it('batchRecommendations trims and batches as expected', async () => {
+	const GETParams = {
+		method: 'GET',
+		headers: {},
+	};
+
+	const batchParams = {
+		siteId: '8uyt2m',
+		lastViewed: [
+			'marnie-runner-2-7x10',
+			'ruby-runner-2-7x10',
+			'abbie-runner-2-7x10',
+			'riley-4x6',
+			'joely-5x8',
+			'helena-4x6',
+			'kwame-4x6',
+			'sadie-4x6',
+			'candice-runner-2-7x10',
+			'esmeray-4x6',
+			'camilla-230x160',
+			'candice-4x6',
+			'sahara-4x6',
+			'dayna-4x6',
+			'moema-4x6',
+		],
+		product: ['marnie-runner-2-7x10'],
+	};
+
+	it('batchRecommendations batches as expected', async () => {
 		let api = new RecommendAPI(new ApiConfiguration({}));
 
-		const GETParams = {
-			method: 'GET',
-			headers: {},
-		};
-
-		const batchParams = {
-			siteId: '8uyt2m',
-			batched: true,
-			lastViewed: [
-				'marnie-runner-2-7x10',
-				'ruby-runner-2-7x10',
-				'abbie-runner-2-7x10',
-				'riley-4x6',
-				'joely-5x8',
-				'helena-4x6',
-				'kwame-4x6',
-				'sadie-4x6',
-				'candice-runner-2-7x10',
-				'esmeray-4x6',
-				'camilla-230x160',
-				'candice-4x6',
-				'sahara-4x6',
-				'dayna-4x6',
-				'moema-4x6',
-			],
-			limits: 20,
-			product: ['marnie-runner-2-7x10'],
-		};
-
-		const GETRequestUrl =
-			'https://8uyt2m.a.searchspring.io/boost/8uyt2m/recommend?tags=similar&tags=crossSell&limits=20&limits=20&siteId=8uyt2m&lastViewed=marnie-runner-2-7x10&lastViewed=ruby-runner-2-7x10&lastViewed=abbie-runner-2-7x10&lastViewed=riley-4x6&lastViewed=joely-5x8&product=marnie-runner-2-7x10';
-
 		let requestMock = jest
 			.spyOn(global.window, 'fetch')
 			.mockImplementation(() => Promise.resolve({ status: 200, json: () => Promise.resolve(mockData.recommend()) } as Response));
 
-		//15 lastViewed trimmed to first 5, & product array changed to single product string
+		const GETRequestUrl =
+			'https://8uyt2m.a.searchspring.io/boost/8uyt2m/recommend?tags=similar&tags=crossSell&limits=20&limits=20&siteId=8uyt2m&lastViewed=marnie-runner-2-7x10&lastViewed=ruby-runner-2-7x10&lastViewed=abbie-runner-2-7x10&lastViewed=riley-4x6&lastViewed=joely-5x8&lastViewed=helena-4x6&lastViewed=kwame-4x6&lastViewed=sadie-4x6&lastViewed=candice-runner-2-7x10&lastViewed=esmeray-4x6&lastViewed=camilla-230x160&lastViewed=candice-4x6&lastViewed=sahara-4x6&lastViewed=dayna-4x6&lastViewed=moema-4x6&product=marnie-runner-2-7x10';
+
+		//product array changed to single product string
 		// @ts-ignore
 		api.batchRecommendations({
 			tags: ['similar'],
+			limits: 20,
+			batched: true,
 			...batchParams,
 		});
 		// @ts-ignore
 		api.batchRecommendations({
 			tags: ['crossSell'],
+			limits: 20,
```

Comment:
It may be useful to have these be non-default values and differ from each other to ensure that the batching sends them in the correct order and with the correct value.